client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -67,6 +73,19 @@ export function getEnhancedTasks(
 
 	const shouldDisplayWarning = displayGlobalStylesWarning || isVideoPressFlowWithUnsupportedPlan;
 
+	const launchpadRecordSignupComplete = () =>
+		recordSignupComplete( {
+			flow,
+			siteId: site?.ID,
+			isNewUser: siteCount <= 1,
```

Comment:
`NB:` I notice we converted siteCount into `0` if it's a `null` or `undefined` value. Would it be easier to check if it's one of those values here to determine  `isNewUser`?

Ex. `isNewUser: siteCount ?? true`

Reply:
Yes, that's simpler. Updated this.

Reply:
I totally borked this logic! Sorry Erick!

Reply:
I believe `Boolean( siteCount )` should do everything we want